insonmnia/structs/order.go

```diff
@@ -98,3 +99,7 @@ func (o *Order) GetType() pb.OrderType {
 func (o *Order) GetByuerID() string {
 	return o.inner.GetByuerID()
 }
+
+func (o *Order) GetDuration() time.Duration {
+	return time.Duration(o.inner.Slot.Duration)
```

Comment:
Inner slot keeps duration in seconds, so use it as `time.Duration(o.inner.Slot.Duration) * time.Second`